src/Section1/style.js

```diff
@@ -1,45 +1,94 @@
-
-import styled from 'styled-components';
-import { ReactComponent as Img } from '../programming 2.svg';
-
-
+import styled from "styled-components";
+import { ReactComponent as Img } from "../programming 2.svg";
 
 export const Container = styled.section`
-
-width: 100%;
-height: 608px;
-background: #FFD66C;
+  position: relative;
+  width: 100%;
+  height: 597px;
+  background: #ffd66c;
+
+  @media (max-width: 768px) {
+    width: 100%;
+    height: 603px;
+  }
+
+  @media (max-width: 376px) {
+    width: 100%;
+    height: 439px;
+  }
 `;
 
 export const Title = styled.h1`
-
-width: 515px;
-height: 167px;
-
-margin-bottom: 302px;
-margin-left: 44px;
-padding-top:128px;
-
-
-font-style: normal;
-font-weight: bold;
-font-size: 74px;
-line-height: 87px;
-
-color: #FFFFFF;
+  width: 515px;
+  height: 167px;
+
+  padding-bottom: 302px;
+  padding-left: 44px;
+  padding-top: 128px;
+
+  font-style: normal;
```

Comment:
If the `font-style` is "normal" and you won't going to change it, you don't need to declare it explicitly.